src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -222,6 +223,16 @@ def load_phabricator(self, diff_phid):
         self.revision = self.api.load_revision(self.phid)
         self.id = self.revision['id']
 
+        # Load build for status updates
+        if settings.build_plan:
+            build, targets = self.api.find_diff_build(self.diff_phid, settings.build_plan)
+            self.build_phid = build['phid']
+            assert len(targets) > 0, 'No build target found'
```

Comment:
Should we assert that there is only one?

Reply:
I'm not sure yet how the API will behave when a revision is updated

Reply:
OK, let's add a warning and then we can check in a while if we can turn it into an assertion (filing a follow-up issue so we don't forget)